app/tests/unittests/test_exception_catching_for_mseed_converter.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from mseed_converter import convert_to_mseed
+from azure.core.exceptions import ResourceNotFoundError
+
+
+class TestMseedConverterExceptionsRaised(unittest.TestCase):
+    def testWrongFilePath(self):
+        wrong_azure_storage_decimated_file_path = "d/d/d/d/oseberg-test.ccs-oseberg.segy"
+        self.assertRaises(ResourceNotFoundError, convert_to_mseed, wrong_azure_storage_decimated_file_path)
```

Comment:
```suggestion
        with self.assertRaises(ResourceNotFoundError):
            convert_to_mseed(wrong_azure_storage_decimated_file_path)
```
I find this to be a bit more readable...